core/src/avm2/value.rs

```diff
@@ -839,10 +839,7 @@ impl<'gc> Value<'gc> {
             }
             Value::Integer(i) => {
                 if *i >= 0 && *i < 10 {
-                    activation
-                        .context
-                        .interner
-                        .get_char(activation.context.gc_context, '0' as u16 + *i as u16)
+                    activation.strings().get_char('0' as u16 + *i as u16)
```

Comment:
For example, this could use `get_ascii_char`

Reply:
I prefer leaving this change for a future PR :)